### ISSUE TITLE
paddlehub 1.8 最后 multi_label_cls_task.predict 推荐的时候怎么不返回概率了

### ISSUE THREAD

**Issue by wa3926** [PaddlePaddle/PaddleFormers#1107]
目前我是这样推荐的最后  res=multi_label_cls_task.predict(data=encoded_data, label_list=label_list) 

返回的是 标签 0 或者1   我想返回标签和对应的概率  请问应该怎么传参   文档里面找不到

**Comment by haoyuying** [comment]
请提供具体的运行模型，环境和脚本，方便我们复现问题，multi_label_cls_task的输入和输出的源码在这里，希望对你有帮助。https://github.com/PaddlePaddle/PaddleHub/blob/release/v1.8/paddlehub/finetune/task/classifier_task.py#L331

**Comment by wa3926** [comment]
> 请提供具体的运行模型，环境和脚本，方便我们复现问题，multi_label_cls_task的输入和输出的源码在这里，希望对你有帮助。https://github.com/PaddlePaddle/PaddleHub/blob/release/v1.8/paddlehub/finetune/task/classifier_task.py#L331



class Toxbdw(MultiLabelDataset):
    """
    The kaggle Toxic dataset:
    https://www.kaggle.com/c/jigsaw-toxic-comment-classification-challenge
    """
    def __init__(self, tokenizer, max_seq_len):
        
        label_list =[这里面20个标签]
        #print("list(dic.values())",list(dic.values()))
        super(Toxbdw, self).__init__(
            base_path=basepath,
            train_file="train.csv",
            dev_file="dev.csv",
            test_file="test.csv",
            train_file_with_header=True,
            dev_file_with_header=True,
            test_file_with_header=True,
            label_file=None,
            label_list=label_list,
            tokenizer=tokenizer,
            max_seq_len=max_seq_len)

    def _read_file(self, input_file, phase=None):
        """Reads a tab separated value file."""
        data = pd.read_csv(input_file, encoding="UTF-8")
        examples = []
        for index, row in data.iterrows():
            guid = row["id"]
            text = row["content"]
            labels = [int(value) for value in row[2:]]
            example = InputExample(guid=guid, label=labels, text_a=text)
            examples.append(example)

        return examples




module = hub.Module(name="ernie_tiny")
inputs, outputs, program = module.context(
    trainable=True, max_seq_len=args.max_seq_len)
dataset =Toxbdw(None,max_seq_len=args.max_seq_len)
label_list = dataset.get_labels()
pooled_output = outputs["pooled_output"]
config = hub.RunConfig(
    use_data_parallel=False,
    use_cuda=args.use_gpu,
    batch_size=args.batch_size,
    checkpoint_dir=args.checkpoint_dir,
    strategy=hub.finetune.strategy.DefaultFinetuneStrategy())
# Define a classfication fine-tune task by PaddleHub's API
multi_label_cls_task = hub.MultiLabelClassifierTask(
    dataset=dataset,
    feature=pooled_output,
    num_classes=dataset.num_labels,
    config=config)
tokenizer = hub.ErnieTinyTokenizer(
    vocab_file=module.get_vocab_path(),
    spm_path=module.get_spm_path(),
    word_dict_path=module.get_word_dict_path())

 data = []
    data.append(replaceStr_key(tanr))
    # Use the appropriate tokenizer to preprocess the data
    # For ernie_tiny, it will do word segmentation to get subword. More details: https://www.jiqizhixin.com/articles/2019-11-06-9
    encoded_data = [
        tokenizer.encode(text=text, max_seq_len=args.max_seq_len)
        for text in data
    ]
    res=multi_label_cls_task.predict(data=encoded_data, label_list=label_list)

最后打印[[{'市水利局': 0}, {'市生态环境局': 1}......]]

**Comment by wa3926** [comment]
![image](https://user-images.githubusercontent.com/53547082/102036110-f9fb9700-3dfc-11eb-9830-a8cda729909a.png)
![image](https://user-images.githubusercontent.com/53547082/102036149-0da6fd80-3dfd-11eb-9a57-36e5415390ca.png)
我看了下源码 我用的是1.8 _按照官网写的 _compatible_mode 这个永远是false     没直接返回 run_states      返回了 self._postprocessing(run_states)   请问该怎么写

**Comment by wa3926** [comment]
无意间找到原因了 base_task.py  预测方法里面 最后 if return_result or not _compatible_mode:
                return self._postprocessing(run_states)
我看其他地方人回 要去掉 or not _compatible_mode   这个bug 在下一个版本会修好  这个是一个bug

**Comment by wa3926** [comment]
已解决
